cmake/Modules_CUDA_fix/upstream/FindCUDA/select_compute_arch.cmake

```diff
@@ -105,8 +105,11 @@ endif()
 
 if(NOT CUDA_VERSION VERSION_LESS "11.8")
   list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Ada")
+  list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Hopper")
   list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "8.9")
+  list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "9.0" "9.0+PTX")
   list(APPEND CUDA_ALL_GPU_ARCHITECTURES "8.9")
+  list(APPEND CUDA_ALL_GPU_ARCHITECTURES "9.0")
 
   if(CUDA_VERSION VERSION_LESS "12.0")
     set(CUDA_LIMIT_GPU_ARCHITECTURE "9.0")
```

Comment:
I think `CUDA_LIMIT_GPU_ARCHITECTURE` should be larger than the latest supported arch.